pkg/app/config.go

```diff
@@ -65,5 +65,12 @@ func validateConfig(c Config) error {
 		}
 		registryName[rc.Name] = true
 	}
+
+	switch role := c.Openshift.SandboxRole; role {
```

Comment:
Can we make this a case insensitive match?

Reply:
Is `switch role := strings.ToLower(c.Openshift.SandboxRole); role {` agreeable?

Reply:
that is exactly what I was thinking